internal/models/log.go

```diff
@@ -0,0 +1,86 @@
+package models
+
+import (
+	"log"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/selfstat"
+)
+
+// PluginConfig contains individualized plugin configuration.
+type PluginConfig struct {
```

Comment:
Won't need this anymore if you switch the telegraf.PluginConfig to a type struct.